### PR TITLE
Show GenAI signal analysis errors in Slack message.

### PR DESCRIPTION
**Pull request by metroid-samus**

### CODE REVIEW

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 1]
```diff
@@ -287,6 +287,14 @@ def create_action_buttons_message(
     return Message(blocks=signal_metadata_blocks).build()["blocks"]
 
 
+def create_signal_metadata_blocks(signal_metadata_blocks: list[Block], message: str) -> list[Block]:
```
Comment (mvilanova) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:290]:
```suggestion
def create_genai_signal_message_metadata_blocks(signal_metadata_blocks: list[Block], message: str) -> list[Block]:
```

Comment (mvilanova) [reply src/dispatch/plugins/dispatch_slack/case/messages.py:290]: I'd make the function name more specific to genai as we include `:magic_wand: *GenAI Alert Analysis` in the section text.

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 2]
```diff
@@ -316,8 +324,8 @@ def create_genai_signal_analysis_message(
     ).first()
 
     if not first_instance_id or not first_instance_signal:
-        log.warning("Unable to generate GenAI signal analysis. No signal instances found.")
```
Comment (mvilanova) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:319]: I'd keep the warnings so we can measure how often this happens.

Comment (metroid-samus) [reply src/dispatch/plugins/dispatch_slack/case/messages.py:319]: Done.

**Review by mvilanova — APPROVED** [review]

**Pull request merged**